internal/evaluator/conftest_evaluator.go

```diff
@@ -279,6 +279,18 @@ func (c conftestEvaluator) Evaluate(ctx context.Context, inputs []string) (Check
 	return results, nil
 }
 
+// Helper function to sort Result lists.
+func SortResults(i, j int, slice []output.Result) bool {
+	// First, sort by shortcode
+	iCode := extractStringFromMetadata(slice[i], "code")
+	jCode := extractStringFromMetadata(slice[j], "code")
+	if iCode == jCode {
+		// If inconclusive, sort by the message
+		return slice[i].Message < slice[j].Message
+	}
+	return iCode < jCode
```

Comment:
I think the sort order should be code, term and then message. Terms are further classifications of codes, e.g. for `test.test_result_unsupported` code there can be a number of `term`s

Reply:
@zregvart How exactly do you mean by `term`s? To use your example of `test.test_result_unsupported`, would the `term` be `test_result_unsupported`?

Reply:
`term` is an additional piece of metadata. It's dynamically added to a policy rule violation on the rego side of things. Here's what it looks like: https://github.com/hacbs-contract/ec-policies/blob/main/policy/release/cve_test.rego#L38-L51

Reply:
Thanks @lcarva!